source/utils/Helper.h

```diff
@@ -30,6 +30,10 @@
 #include <vector>
 #include <cstdint>
 
+#ifdef _MSC_VER
+    #define snprintf _snprintf
```

Comment:
When I have this kind of existential issue, I usually look at what is done in references sources. And IMHO, Ogre and CEGUI are good references. If they do the same, I would go for doing the same, too.